www/assets/js/db-handler.js

```diff
@@ -223,11 +223,11 @@ var dbHandler = {
 
             store.openCursor().onsuccess = function(event) {
               let cursor = event.target.result;
-              if (cursor) {
+              if (cursor && cursor.value && cursor.value.weight) {
```

Comment:
Hi @davidhealey, the `weight` property appears to be always set, but I'm not sure if that property holds. ALso, is there an easy way to test this repo?

Reply:
There shouldn't be a weight goal at all in the new version, since it isn't used for anything. The weight log is now part of the diary and exists in the stats object.

> Also, is there an easy way to test this repo?

That depends on what you mean by easy. I do 99% of my development in Chromium and only use an actual Android device for testing and debugging.

Reply:
Yes, I wasn't clear. I meant I don't know if there is a need for  the two checks I added... What do you think?
I'll try to run chromium with the inspection feature then.

Reply:
Oh I see, I can't remember exactly what values are there so I'll need to test it and monitor the output to know for sure.

Reply:
I don't know which value is there since the beginning (if you never set a weight). But if you have set it at least once, I think this value is copied to the next day until a new one is inserted. But I'll let you know if I'm able to test it

Reply:
I looked into this today and fixed a few things with the importing/upgrading so we should be good to go now